http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/http2/MultiplexedChannelRecord.java

```diff
@@ -175,6 +175,17 @@ void handleGoAway(int lastStreamId, GoAwayException exception) {
         });
     }
 
+    /**
+     * Prevent new streams from being acquired from the existing connection.
+     */
+    void closeToNew() {
```

Comment:
Nit: closeToNewStreams()? The other methods refer to them as 'streams', so that'd be more consistent.

Reply:
Sounds good. Updated